examples/cpp/flow_control/PublisherApp.cpp

```diff
@@ -214,7 +220,7 @@ void PublisherApp::run()
 
 bool PublisherApp::publish(
         DataWriter* writer_,
-        FlowControl msg)
+        FlowControl& msg)
```

Comment:
It seems that the thread's stack size that windows reserve is lower than the actual required. It seems that avoiding the stack-copy of the FlowControl msg between the `run()` and `publish()` methods solves the issue as it makes it to fit in the stack-size reserved in windows, for me it is okay.

Another way of solving it would be adding these lines 
```
if(WIN32)
    set_target_properties(flow_control PROPERTIES LINK_FLAGS /STACK:4194304)
endif()

```
In the `CMakeLists.txt:50` of the `flow_control` example. I find hard to decide between the two options. Probably just avoiding the copy is good enough.